scripts/view/projectView.js

```diff
@@ -16,17 +16,19 @@
   projectView.handleCategoryFilter = function() {
     $('#category-filter').on('change', function() {
       if ($(this).val()) {
-        $('article').hide();
+        $('#projects article').hide();
         $('article[data-category="' + $(this).val() + '"]').fadeIn();
+        page('projects/' + $(this).val().replace(/\W+/g, '+'));
```

Comment:
I think this is your problem for the endlessly multiplying projects. I think I forward slash before 'projects/' would help. So page('/projects/')[...] might do something useful.